spec/command_line_ui_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'command_line_ui'
+require 'board'
+require 'player_symbols'
+require 'player_symbols'
+
+RSpec.describe CommandLineUI do
+
+  before (:each) do
+    @reader_spy = instance_double("PromptReader")
+    @writer_spy = instance_double("PromptWriter")
+  end
+
+  it "reads move from user" do
+    expect(@writer_spy).to receive(:ask_for_next_move)
+    expect(@writer_spy).to receive(:show_board)
+    expect(@reader_spy).to receive(:get_input).and_return("1")
+
+    command_line_ui = CommandLineUI.new(@writer_spy, @reader_spy)
+
+    expect(command_line_ui.get_move_from_player(Board.new)).to be 1 
+  end
+
+  it "reprompts move from user when an alpha character entered" do
+    expect(@writer_spy).to receive(:ask_for_next_move).twice
```

Comment:
@jsuchy I agree that the tests did have a lot of setup. I paired with @ukutaht this afternoon and he showed me that when declaring the spies, I can use `as_null_object` so that not every single method on the spy has to be checked. 

I have also now broken the tests down into more specific tests, testing one thing, rather than verifying lots of method calls in a single test.  I think the tests now look more focussed.